### ISSUE TITLE
[Feature Request] `Pressable` indicators

### ISSUE THREAD

**Issue by utkarsh1097** [kolking/react-native-page-indicator#20]
For certain types of indicators (such as "bead"), making them clickable/pressable will help navigating from page 'i' to page 'j' easier.

**Comment by kolking** [comment]
Thank you very much for writing the issue! Nevertheless, I'm quite sceptical about this. I believe that in the majority of use cases the indicators are too small to precisely tap, so for me this sounds like an unnecessary complication.

**Comment by utkarsh1097** [comment]
Thanks for your reply!

I think there are some use cases (for example, the one I explained above), and it can be implemented as a boolean flag (default to false for the reasons you mention). 

As far as the implementation detail goes, I think that `IndicatorBeads` can use `Pressable` based on whether the boolean flag is set or not. It can be the user's responsibility to pass the `onPress` callback that manages the state of the active page. (i.e., tapping on the indicator for page 'i' should set the active page to 'i')

**Comment by enkyuan** [comment]
on another note, how would i be able to implement a button (that's tappable) to navigate between the pages? working on that right as part of some onboarding components but it's not looking great to say the least.

**Comment by kolking** [comment]
@enkyuan as far as I understood, you want to have a "Go next" button inside a page. If so, you need to use [`scrollTo()`](https://reactnative.dev/docs/scrollview#scrollto) method to move your ScrollView / FlatList to the new page.

**Comment by enkyuan** [comment]
@kolking thanks for the response; i'll look into it :)

**Comment by enkyuan** [comment]
btw, do you mind if i open a separate issue regarding implementation on that of that nav button? been working on it for a bit and i'm still not that familiar with the react's animated api.

**Comment by kolking** [comment]
@enkyuan sure I don't mind. Just note that your issue is not related with my library (and with the Animated API as well).

**Comment by enkyuan** [comment]
@kolking yeah i know. just asking for some guidance is all. i'll clear the issue immediately after i figure things out.
